# Synthetic Datasets

## CINECA_synthetic_cohort_EUROPE_UK1
This dataset consists of 2521 samples which have genetic data based on [1000 Genomes data](https://www.nature.com/articles/nature15393), and synthetic subject 
attributes and phenotypic data derived from [UKBiobank](https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1001779). These data were initially derived using the [TOFU](https://github.com/spiros/tofu) tool, which 
generates randomly generated values based on the UKBiobank data dictionary. Categorical values were randomly generated 
based on the data dictionary, continuous variables generated based on the distribution of values reported by the UK 
Biobank showcase, and date / time values were random. Additionally we split the phenotypes and attributes into 4 main 
classes - general, cancer, diabetes mellitus, and cardiac. We assigned the general attributes to all the samples, and 
the cardiac / diabetes mellitus / cancer attributes to a proportion of the total samples. Once the initial set of 
phenotypes and attributes were generated, the data was checked for consistency and where possible dependent attributes 
were calculated from the independent variables generated by TOFU. For example, BMI was calculated from height and 
weight data, and age at death generated by date of death and date of birth. These data were then loaded to the 
development instance of [Biosamples](https://www.ebi.ac.uk/biosamples/ ) which accessioned each of the samples. 

The genetic data are derived from the [1000 Genomes Phase 3 release](https://www.internationalgenome.org/category/phase-3/). The genotype data consists of a single joint call 
vcf file with call genotypes for all 2504 samples, plus bed, bim, fam, and nosex files generated via plink for these 
samples and genotypes. The genotype data has had a variety of errors introduced to mimic real data and as a test for 
quality control pipelines. These include gender mismatches, ethnic background mislabelling and low call rates for a 
randomly chosen subset of sample data as well as deviations from Hardy Weinberg equilibrium. 
Additionally 40 samples have raw genetic data available in the form of both bam and cram 
files, including unmapped data. The gender of the samples in the 1000 genomes data has been matched to the synthetic 
phenotypic data generated for these samples. The genetic data was then linked to the synthetic data in BioSamples, and 
submitted to [EGA](https://ega-archive.org/). 

### Data Access
#### EGA

#### BioSamples
Phenotypic data for CINECA_synthetic_cohort_EUROPE_UK1 are publicly available at the dev instance of BioSamples 
database. At the moment there are 2504 synthetic sample entries adhering to the UKB data distribution. They all have a 
`project` attribute set to `UKB_SYNTHETIC_DATA` and therefore can be easily filtered out by applying the filter 
[`attr:project:UKB_SYNTHETIC_DATA`](https://wwwdev.ebi.ac.uk/biosamples/samples?filter=attr:project:UKB_SYNTHETIC_DATA) 
on the BioSamples.


### Links
- Description: [doc](), [ppt]()
- [Google Drive](https://drive.google.com/drive/u/1/folders/1uEy6C_owT6Lh1gBWHs-MvA4nS_9TAdLK)
- [BioSamples](https://wwwdev.ebi.ac.uk/biosamples/samples?filter=attr:project:UKB_SYNTHETIC_DATA)
- [EGA](https://ega-archive.org/datasets/EGAD00001006673)


## CINECA_synthetic_cohort_Africa_H3ABioNet1

## CINECA_synthetic_cohort_CH_SIB

## CINECA_synthetic_cohort_Canada_CHILD1
